build/parser.mjs

```
@@ -1038,20 +1038,45 @@ class Parser {
+      }
+    };
+
+    if (hasRelicDrops) {
+      item.components.forEach((component) =>
+        component.drops?.forEach((drop) => {
+          drop.name = drop.location;
+
+          addRelic(drop, true);
+
+          // only need for the func to work
+          delete drop.name;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)